### PR TITLE
Use point / size / rectangle structures consistently in API docs

### PR DESCRIPTION
**Pull request by miniak**
I've added `Point` and `Size` types to avoid duplicate types being generated. I've also fixed a few places where `Rectangle` was not being used.

Related to https://github.com/electron/electron/issues/4875

### CODE REVIEW

File: docs/api/browser-window.md [hunk 1]
```diff
@@ -725,7 +723,7 @@ Resizes and moves the window to the supplied bounds
 
 #### `win.getBounds()`
 
-Returns [`Rectangle`](structures/rectangle.md)
+Returns [Rectangle](structures/rectangle.md)
```
Comment (MarshallOfSound) [inline docs/api/browser-window.md]: "Rectangle" needs to be wrapped in code backticks.

```markdown
[`Rectangle`](structure/rectangle.md)
```

Comment (MarshallOfSound) [inline docs/api/browser-window.md]: Same for all the other return types 👍

Comment (miniak) [inline docs/api/browser-window.md]: oh, I'll change it back to backticks. Does this only apply to return values?

Comment (MarshallOfSound) [inline docs/api/browser-window.md]: Yeah, only return values

Comment (miniak) [inline docs/api/browser-window.md]: done

**Comment by kevinsawicki** [comment]
Thanks @miniak for improving these 👍 🚀

**Pull request merged**